docs/tool_jcmd.md

```diff
@@ -39,6 +39,10 @@ Where:
 
 - `<vmid>` is the Attach API virtual machine identifier for the Java&trade; VM process. This ID is often, but not always, the same as the operating system *process ID*. One example where the ID might be different is if you specified the system property `-Dcom.ibm.tools.attach.id` when you started the process. You can use the [`jps`](tool_jps.md) command to find the VMID.
 
+You can also specify the full or partial Java process display name instead of the VMID. The `jcmd` tool finds the corresponding VMID of the display name and runs the `jcmd` command.
```

Comment:
What is a Java process display name, is this defined somewhere?

Reply:
Jason pls help clear this up. The linked topic https://eclipse.dev/openj9/docs/dcomibmtoolsattachdisplayname/ could use more explanation of what the display name is, what is the default, what it's used for.

Reply:
`Jcmd` or `Jcmd -l` lists current java processes recognized by this utility, the list comes with two columns, one is VMID which usually is the process id, and the other is the display name.

The display name for a virtual machine could be specified via a system property `com.ibm.tools.attach.displayName` as per https://eclipse.dev/openj9/docs/dcomibmtoolsattachdisplayname/, otherwise it is a value from the system property [sun.java.command](https://github.com/eclipse-openj9/openj9/blob/3b697e1318b4e29d727e3dfb3b29a07b793eaa92/jcl/src/java.base/share/classes/openj9/internal/tools/attach/target/AttachHandler.java#L290).

Reply:
It seems a well known system property:
* https://cr.openjdk.org/~lbourges/marlin/j2dBench_reports/html_ductus_marlin_071/System_Properties.html
* https://github.com/spring-projects/spring-boot/issues/12796
* https://bugs.eclipse.org/bugs/show_bug.cgi?id=491364

Reply:
But can you find any official documentation for sun.java.command? Just because it's known to some doesn't mean we should refer to it as if it's a known thing to everyone.

Reply:
Replace it with `the main class name along with the application arguments.`

Reply:
> `Jcmd` or `Jcmd -l` lists current java processes recognized by this utility, the list comes with two columns, one is VMID which usually is the process id, and the other is the display name.
> 
> The display name for a virtual machine could be specified via a system property `com.ibm.tools.attach.displayName` as per https://eclipse.dev/openj9/docs/dcomibmtoolsattachdisplayname/, otherwise it is the main class name along with the application arguments.

@JasonFengJ9  - Is it "The display name for a virtual machine" or "The display name for the Java VM process"?

Reply:
> Is it "The display name for a virtual machine" or "The display name for the Java VM process", or "The display name for the Attach API virtual machine" or "The display name for the target virtual machine on which the jcmd command must run."?

When `Jcmd` or other AttachAPI utilities list the currently running Java processes, the display name refers to the target Java process that could be attached by `Jcmd`.

> is -l, one of the options available for jcmd in addition to -J and -h?

Correct.

> Also, do both jcmd and jcmd -l list the current java processes recognized by this utility? i.e. even if we don't specify the -l option, will it still list the current java processes?

Yes, `-l` is the default option.

Reply:
Is the following correct? Especially this part `jcmd [pid | main-class] command`. I saw this mentioned in https://github.com/eclipse-openj9/openj9/issues/19045#issuecomment-1971174095

> The display name for a target VM is specified either through a system property [`com.ibm.tools.attach.displayName`](dcomibmtoolsattachdisplayname.md), or by specifying the main class name along with the application arguments - `jcmd [pid | main-class] command`.

Reply:
The `main-class` (along with application arguments) are the default display name if no system property [com.ibm.tools.attach.displayName](https://github.com/eclipse-openj9/openj9-docs/pull/dcomibmtoolsattachdisplayname.md) was set.

Suggest changing to something like following:
_The display name for a target VM is specified through a system property [com.ibm.tools.attach.displayName](https://github.com/eclipse-openj9/openj9-docs/pull/dcomibmtoolsattachdisplayname.md), or by default the main class name along with the application arguments - jcmd [pid | main-class] command._